src/Renderer/Shader/TileFS.glsl

```diff
@@ -122,21 +122,22 @@ void main() {
                             textureIndex,
                             projWGS84 ? vUv_WGS84 : uvPM);
 
-                        if (layerColor.a > 0.0) {
+                        bool isVisible = layerColor.a > 0.0;
+                        // if first texture is transparent, we need to force blending with first background color (CWhite)
+                        if ((isVisible && paramsA.w > 0.0) || !validTexture) {
```

Comment:
Are you sure you still need this `!validTexture` test now that `diffuseColor` is properly initialized to `noTextureColor`?

Reply:
yes, I removed